pagetoc.css

```diff
@@ -0,0 +1,84 @@
+/* Inspired by https://github.com/JorelAli/mdBook-pagetoc/tree/98ee241 (under WTFPL) */
```

Comment:
Question [LICENSE 1/2]: hm, this is not Apache 2.0 / MIT, would this be a licensing problem?